Revert/Number.java

```diff
@@ -0,0 +1,81 @@
+import java.util.*;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.text.*;
+
+public class Number {

```

Comment:
Wydaje mi się, że to nie do końca trafna nazwa, ponieważ klasa Number jest już w bibliotece standardowej Javy